glean-core/src/lib.rs

```diff
@@ -678,7 +678,7 @@ impl Glean {
     /// Register a new [`PingType`](metrics/struct.PingType.html).
     pub fn register_ping_type(&mut self, ping: &PingType) {
         if self.ping_registry.contains_key(&ping.name) {
-            log::debug!("Duplicate ping named '{}'", ping.name)
+            log::error!("Duplicate ping named '{}'", ping.name)
```

Comment:
We only [just recently](https://github.com/mozilla/glean/commit/b61b60d4977cffca57b09a32190638b5b163bc05) dropped this log level, as it was surfaced in Sentry on iOS, but is of no help.

```suggestion
            log::debug!("Duplicate ping named '{}'", ping.name)
```